.github/workflows/shared_workflow.yml

```
@@ -180,29 +182,12 @@ jobs:
     name: API tests
     strategy:
       matrix:
-        nextcloudVersion: [ stable26, stable27, stable28, stable29, stable30 ]
+        nextcloudVersion: [ stable30 ]
         phpVersionMajor: [ 8 ]
-        phpVersionMinor: [ 0, 1, 2, 3 ]
+        phpVersionMinor: [ 3 ]
```

```suggestion
        phpVersionMinor: [ 3 ]
```
Would it be necessary to run API tests for other versions of Nextcloud and PHP as well?
Is what we have here is sufficient?@SagarGi @individual-it

---

I completed the fix and added the before and after list to the PR description